From 17562259fb6684f351a70b98b4f3b438927ba1aa Mon Sep 17 00:00:00 2001
From: Peter Seiderer <ps.report@gmx.net>
Date: Mon, 11 Jul 2016 22:58:31 +0200
Subject: [PATCH] Fix musl compile.

Include proper config.h (via log4cpp/Portability.hh) to fix the
evaluate of LOG4CPP_HAVE_STDINT_H in the file tests/Clock.hh

Fixes:

  Clock.hh:17:13: error: 'int64_t' does not name a type
       typedef int64_t usec_t;

Signed-off-by: Peter Seiderer <ps.report@gmx.net>
---
 log4cpp/tests/Clock.hh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/log4cpp/tests/Clock.hh b/log4cpp/tests/Clock.hh
index 4bf0ca4..45c6467 100644
--- a/log4cpp/tests/Clock.hh
+++ b/log4cpp/tests/Clock.hh
@@ -6,6 +6,8 @@
 #ifndef __CLOCK_H
 #define __CLOCK_H
 
+#include <log4cpp/Portability.hh>
+
 #ifdef LOG4CPP_HAVE_STDINT_H
 #include <stdint.h>
 #endif // LOG4CPP_HAVE_STDINT_H
-- 
2.8.1

